### DISCUSSION TITLE
Need help...Samsung Galaxy Watch 6 40???

### DISCUSSION THREAD

**Discussion by Luckyleft** [trupici/G-Watch-Wear#36] [General]
Hello all...I need some help.

I have a Samsung Galaxy 6 size 44...and a Dexcom G7. I have G-Watch Wear working perfectly!

My wife has been using the Dexcom 6 app on her Samsung Galaxy 6 watch ; however she has a size 40...not 44.  She recently switched to the Dexcom G7. I went to install G-Watch Wear to match mine. We also both have the Samsung S23 FE phone.  Everything exactly the same except the watch size...40 vs 44.

I have tried multiple ways to get her watch working with her G7; however I cannot for the life of me get the G-Watch Wear app/faces to show on her watch.  It's fine on the phone.  I've un-installed, updated, everything I can think of.

I didn't find anything in the installation documentation, but does ANYONE have this working on a Galaxy 6 watch size 40???

If so....please help me.

Thanks!!!

**Comment by Cleboost** [comment]
Hi there! We'd have to see what version of wear os is on the watch, maybe it's not up to date or just not compatible with the build version.

**Reply by Luckyleft** [reply]
Thanks for the reply. WEAR OS 4...all updates applied.  Everything is the same as my watch other than size.

I looked at every setting on each of our phones, there is one difference.  Her phone is the "unlocked" model where mine is associated with a carrier.  Should have no impact.  We have a Samsung Galaxy S23FE

One other weird difference...I've un-installed and reinstalled the g-watch wear app 100 times.  On her phone...it never puts the watch face in the WEAR-->Watch Faces-->Downloaded faces area.  If I go back to G Play Store...it thinks the watch face is installed. So weird!  I would love to get this working for her.

**Reply by Cleboost** [reply]
Whether or not the phone is linked to an operator has no impact on how the application is normally used. I don't think the size of the watch is a problem unless the application is in its code in minimum (I'll check). If you're determined, you can try syncing your wife's watch to your phone and reinstalling the app. We'll see whether it's coming from her watch or her phone. The only thing that intrigues me is the fact that the dial is seen as installed but isn't... To be continued

**Reply by Luckyleft** [reply]
SOLVED

So...I removed wife's watch from her phone, and removed mine. I added her watch to my phone and reset her watch.  I couldn't seem to get the app to install the watch face to her watch. Next I restored a backup of my watch to her watch (which had the G-Wear faces).  It worked!  Next I transfered her watch to her phone...restored mine...and all is well.  Both watches now working.

Thanks for your help!!

**Comment by Mongoose69** [comment]
I have a S23FE and Galaxy Watch 5 instead of 6, size 40 and it works fine, albeit I don't have Dexcom, just Libre 2

**Reply by Luckyleft** [reply]
Thanks!
